crates/sol-types/Cargo.toml

```diff
@@ -35,4 +35,5 @@ trybuild = "1.0"
 [features]
 default = ["std"]
 std = ["alloy-primitives/std", "hex/std", "serde?/std"]
+sol-json = ["alloy-sol-macro/json"]
```

Comment:
should this be a default feature?

Reply:
I don't think so, it does bring in quite a bit of extra deps and it should be avoided when possible